### PR TITLE
Translate some code, comments and docstrings

### PR DESCRIPTION
**Pull request by y-martinez**
This PR partially Close: #184 

This PR adds:

- Translate some code, comments, and docstrings.

### CODE REVIEW

File: utilities/buscarEnBloque.py [hunk 1]
```diff
@@ -29,27 +29,27 @@
 def main(manager, nom_item, verbose):
     info = manager.get_item(nom_item)
     if info is None:
-        print("No se encontró el item")
+        print("Item not found")
     else:
-        print("Largo item devuelto", len(info))
+        print("Long item returned", len(info))
```
Comment (facundobatista) [inline utilities/buscarEnBloque.py:34]:
```suggestion
        print("Returned item length", len(info))
```

File: utilities/makeLista.py [hunk 2]
```diff
@@ -22,13 +22,13 @@
 import sys
 
 usage = """
-Usar: makeLista.py <directorio>
+Use: makeLista.py <directory>
 
-   donde el directorio es donde está descomprimida la wikipedia
+    whose directory is where is decompressed the wikipedia
```
Comment (facundobatista) [inline utilities/makeLista.py:27]:
```suggestion
    directorye where the wikipedia is decompressed
```

File: utilities/verComprimido.py [hunk 3]
```diff
@@ -31,15 +31,15 @@
...
 def main(fname, a_extraer):
     fsize = os.stat(fname).st_size
-    print("Mostrando info del archivo %r (tamaño: %d bytes)" % (fname, fsize))
+    print("Showing info of the file %r (size: %d bytes)" % (fname, fsize))
     if fname.endswith('.cdi'):
         c = BloqueImagenes(fname)
     else:
         c = Comprimido(fname)
-    print("Del header (%d bytes): %d archivos en total" % (c.header_size, len(c.header)))
+    print("The header (%d bytes): %d files in total" % (c.header_size, len(c.header)))
```
Comment (facundobatista) [inline utilities/verComprimido.py:39]:
```suggestion
    print("From the header (%d bytes): %d files in total" % (c.header_size, len(c.header)))
```

File: utilities/verComprimido.py [hunk 4]
```diff
@@ -53,27 +53,27 @@ def main(fname, a_extraer):
...
 
 
 if __name__ == "__main__":
     if len(sys.argv) < 2:
-        print("Usar:  verComprimido.py <comprimido> [archivo [...]]")
-        print("           donde el archivo comprimido es un .cdp / .cdi")
-        print("           opcionalmente, se pueden pasar archivos a extraer")
+        print("Use:  verComprimido.py <compressed> [file [...]]")
+        print("           where the compressed file is a .cdp / .cdi")
+        print("           optionally, can pass files to extract")
```
Comment (facundobatista) [inline utilities/verComprimido.py:76]:
```suggestion
        print("           optionally, files to extract can be indicated")
```

**Review by facundobatista — CHANGES_REQUESTED** [review]
Awesome, thanks for this work! I included some slight changes. Thanks again!

**Comment by y-martinez** [comment]
Hi @facundobatista I made the changes requested, could you review it  please :smiley:

File: src/armado/to3dirs.py [hunk 5]
```diff
@@ -109,6 +109,6 @@ def get_path_file(page):
 
 
 def from_path(path):
-    """ Quita los 3 dirs del path """
+    """Remove the 3 directories of the path"""
```
Comment (facundobatista) [inline src/armado/to3dirs.py:112]: Finish the sentence with a `.`.

File: src/generate.py [hunk 6]
```diff
@@ -227,7 +226,7 @@ def prepare_temporary_dirs(process_articles):
     dtemp = config.DIR_TEMP
     if os.path.exists(dtemp):
         if not process_articles:
-            # preparamos paths y vemos que todo esté ok
+            # we prepare paths and verify that everything is ok
```
Comment (facundobatista) [inline src/generate.py:229]: remove the "we" when starting these comments

File: src/web/destacados.py [hunk 7]
```diff
@@ -45,6 +45,7 @@ def __init__(self, article_manager, debug=False, verbose=False):
         self._iter = itertools.cycle(self.destacados)
 
     def get_destacado(self):
+        """Return a destacado randomly... eventually"""
```
Comment (facundobatista) [inline src/web/destacados.py:48]: You have to remove the Spanish docstring.

**Review by facundobatista — CHANGES_REQUESTED** [review]
Great work!!! Thanks!! I added some details inline, thanks again!

File: utilities/benchmarkIndice.py [hunk 8]
```diff
@@ -60,47 +60,47 @@ def usoMemoria():
...
+    memdesp = memoryUsage()
 
-    print("               ocupa memoria:  %d KB" % (memdesp - memant))
+    print("               occupy memory:  %d KB" % (memdesp - memant))
 
-    with Timer("Listado completo palabras"):
+    with Timer("Full list of words"):
         palabras = [x.decode("utf8") for x in indice.listado_palabras()]
-    print("               cant palabras:", len(palabras))
+    print("               cant words:", len(palabras))
```
[the remark this answers is not in the document]
Comment (y-martinez) [reply utilities/benchmarkIndice.py:74]: In this case, a valid translation can be `numbers of words` or maybe `quantity of words` what do you think?

File: utilities/buscarEnBloque.py [hunk 9]
```diff
@@ -29,27 +29,27 @@
...
     parser.set_usage(msg)
     parser.add_option(
         "-v", "--verbose", action="store_true", dest="verbose",
-        help="muestra info de lo que va haciendo")
+        help="shows info of what it is doing")
 
     parser.add_option(
         "-i", "--image", action="store_true", dest="image",
-        help=u"busca en imagenes (busca artículos por default)")
+        help=u"search in images (search for articles by default)")
```
[the remark this answers is not in the document]
Comment (y-martinez) [reply utilities/buscarEnBloque.py:52]: I have doubts about this, I think that a possible solution is to translate
> search in images (searches articles by default)

or maybe 

> searches in images (shows articles by default)

**Comment by facundobatista** [comment]
> Hi @facundobatista I made the changes requested, could you review it please smiley

Let me know please when you can check those details I marked, thanks!

**Comment by y-martinez** [comment]
> > Hi @facundobatista I made the changes requested, could you review it please smiley
> 
> Let me know please when you can check those details I marked, thanks!

Yes I did make the changes but I have a few doubts so I asked you two questions above, can you see them?

File: utilities/benchmarkIndice.py (hunk 8 above)
[the remark this answers is not in the document]
Comment (facundobatista) [reply utilities/benchmarkIndice.py]: quantity of words

File: utilities/buscarEnBloque.py (hunk 9 above)
[the remark this answers is not in the document]
Comment (facundobatista) [reply utilities/buscarEnBloque.py]: My comment is about the consistency between the help messages of the different parameters. Make them all "imperative".

**Comment by facundobatista** [comment]
I answered the questions, but found that lot of details I marked are not fixed.

**Comment by y-martinez** [comment]
> I answered the questions, but found that lot of details I marked are not fixed.

Ohh gosh I was stupid, I forgot that I stashed those changes, and I didn't commit them, I already made the changes

**Review by facundobatista — APPROVED** [review]
Great, thanks!

**Pull request merged**